invenio_app_ils/records/api.py

```diff
@@ -597,16 +599,18 @@ def validate_create(cls, data):
 
     def validate_patch(self, ops):
         """Validate document request data."""
-        for op in ops:
-            if "state" in op["path"] and op["value"] == "FULFILLED":
-                # Ensure there is also a document_pid in the ops
-                found_document_pid = False
-                for op in ops:
-                    if "document_pid" in op["path"]:
-                        self._validate_fulfilled(document_pid=op["value"])
-                        found_document_pid = True
-                if not found_document_pid:
-                    self._validate_fulfilled(document_pid=None)
+        # Check if state == 'FULFILLED' has been passed
+        fulfilled_state = any(
+            op['path'] == '/state' and op['value'] == 'FULFILLED'
```

Comment:
maybe not valid as a comment here but can we name the `ops` variable with some more readable name?

Reply:
Some suggestions: `operations` or `patches`. Using `patches` might make it more clear actually since each patch has an operation.